requirements.txt

```diff
@@ -1 +1,2 @@
 # Project dependencies. Please keep this list in alphabetical order.
+flask==2.2.2
```

Comment:
let's add a new line after "header" of this file and a the end of the file as well